### ISSUE TITLE
Custom library folder for each Stable, Daily and experimental

### ISSUE THREAD

**Issue by aditiapratama** [DotBow/Blender-Launcher#13]
**Is your feature request related to a problem? Please describe.**
The blender is already installed and I just want to use Blender Launcher to update existing, in a designated folder.

**Describe the solution you'd like**
Option to enable custom folder for each Stable, Daily, and Experimental. I don't mind having a library folder though.

**Comment by DotBow** [comment]
Hi!
As far as I understand you want to have only one version of blender and update it...
I see such function useful for stable releases, but since daily very unstable and mostly for testing purposes there is no any good reason to have one at a time.
Anyway, right now the core architecture doesn't allow to do it, and structure of folders is an abstract layer that user should not think about. 
If you could provide more detailed and real use case then it may be easier to  consider about implementation.

**Comment by aditiapratama** [comment]
> As far as I understand you want to have only one version of blender and update it...
Nope, I have multiple version of Blender installed...the reason I request this is because I want to merge/transition my current setup, I already put blender in `$PATH` and I'll have BL to manage the Stable, Daily and Experimental in my preferred folder. I know I can use symlink as well...but just in case if there is an UI to do that would be great :)
